src/Directory.Build.props

```diff
@@ -24,15 +24,24 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>../../assets/Destructurama.snk</AssemblyOriginatorKeyFile>
+    <NoWarn>$(NoWarn);Proj0003;Proj0010;Proj0028;Proj0200;Proj0201;Proj0205;Proj0206;Proj0208;Proj0213;Proj0214;Proj0215;Proj0216;Proj0240;Proj0243;Proj0400;Proj0800;Proj1001;Proj1003;</NoWarn>
```

Comment:
If you want to disable these, you better use a `.globalconfig` file.

Reply:
I prefer to use editorconfig for all projects I participate of. For now it's just a first attempt to use analyzer. I evaluate pros and cons. My goal is to find reasonable set of diagnostics and those ones that have a little value.

Reply:
> I prefer to use editorconfig for all projects I participate of. For now it's just a first attempt to use analyzer. I evaluate pros and cons. My goal is to find reasonable set of diagnostics and those ones that have a little value.

That is true. However, MS itself states that the configuration of files should be specified in the `.globalconfig` instead of the `.editorconfig`. Some rules (including the of .NET project file analyzers) can not be configured in `.editorconfig`, but can be configured in `.globalconfig`.

Reply:
> MS itself states that the configuration of files should be specified in the .globalconfig instead of the .editorconfig

Could you please point to the relevant documentation?

Reply:
The link can be found here:

https://dotnet-project-file-analyzers.github.io/general/configuration.html